exercises/pig-latin/src/test/java/PigLatinTranslatorTest.java

```diff
@@ -53,6 +58,15 @@
                 // Xr is treated like a single vowel
                 {"xray", "xrayay"},
 
+                // 'Y' is treated like a consonant at the beginning of a word
+                {"yellow", "ellowyay"},
+
+                // 'Y' is treated like a vowel at the end of a consonant cluster
+                {"rhythm", "hythmray"},
```

Comment:
This should map to `"ythmrhay"`. If the build doesn't pass when you do that then you need to update the reference solution so that it maps it correctly. Does that make sense? 🙂

Reply:
Alright, thanks! Two questions:
 1) Is it feasible to set up the same TravisCI builds in my fork (I'd rather run the tests online than 
     locally)?
 2) Does updating the reference solution mean changing the canonical data?

Thank you!